website/templates/public/register.mako

```diff
@@ -49,6 +49,18 @@
         </div>
     %endif
 
+    %if campaign == "bitss-preprints":
```

Comment:
This should be much more modular. However, we decide just to "patch" this page since it will eventually go away (moved to CAS) 2-3 weeks after Django-OSF goes alive.